schema.sql

```diff
@@ -12,5 +12,13 @@ CREATE TABLE IF NOT EXISTS companies (
   `company_description` TEXT NOT NULL,
   `company_details` TEXT NOT NULL,
   `company_rating` TEXT NOT NULL,
-  `company_logo` TEXT NOT NULL
+  `company_logo` TEXT NOT NULL,
+  `company_adress` TEXT NOT NULL UNIQUE,
+  `category_id` INTEGER
+);
+
+
+CREATE TABLE IF NOT EXISTS category (
+  `category_id` INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
You don't have to repeat the table name in the field names, use `id`, `type`. 
But of course, if you have a foreign key to another table, then the name of the field should be `<other_table_name>_id`